core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.spec.ts

```diff
@@ -496,6 +496,65 @@ describe('Translation Modal Component', () => {
       });
     });
 
+    describe('when original text has paragraphs with spaces', () => {
+      it('should exclude paragraphs with spaces', () => {
+        component.textToTranslate = '<p>First para</p><p>&nbsp;</p>';
+
+        const filteredText = component.modifyContentToValidate(
+          component.textToTranslate);
+
+        expect(filteredText).toBe('<p>First para</p>');
+      });
+    });
+
+    describe('when translated content have elements with missing child' +
```

Comment:
Nit: I think it should be "has" elements as "content" is singular? Here and below.

Reply:
Done